Gruntfile.js

```diff
@@ -80,6 +81,23 @@ module.exports = function(grunt) {
         finRepl.startRepl();
     });
 
+    grunt.registerTask('kill-processes', function() {
+
+          // looks up the process by the program name
+          ps.lookup({
+              command: 'openfin.exe'
+          }, function(err, processList) {
+              if (err) {
+
+                  // if error occurs then thow error
+                  throw new Error( err );
+              }
+
+              // return the process ids of each process found
+              processList.forEach( i => ps.kill(i.pid))
```

Comment:
extra space, missing semicolon and confusing argument name (`i` is usually used for indices) `processList.forEach(process => ps.kill(process.pid));`